src/pages/Signup/index.js

```diff
@@ -61,47 +68,73 @@ const Signup = () => {
   }
 
   return (
-    <div className="form">
-      <form onSubmit={handleSubmit(onSubmit)} noValidate>
-        <h1>{t('signup.title')}</h1>
-        <label htmlFor="email">{t('signup.labels.email')}</label>
-        <Input
-          register={register}
-          type="email"
-          name="email"
-          error={errors.email}
-          handleFocus={handleFocus}
-        />
-
-        <label htmlFor="password">{t('signup.labels.password')}</label>
-        <Input
-          register={register}
-          type="password"
-          name="password"
-          error={errors.password}
-          handleFocus={handleFocus}
-        />
-
-        <label htmlFor="password">{t('signup.labels.passwordConfirmation')}</label>
-        <Input
-          register={register}
-          type="password"
-          name="passwordConfirmation"
-          error={errors.passwordConfirmation}
-          handleFocus={handleFocus}
-        />
-
-        {error && error.data && (
-          <p className="error-message">{error.data.errors?.full_messages[0]}</p>
-        )}
-
-        <div className="button-container">
-          <Button type="submit" disabled={isLoading}>
-            {t('signup.title')}
-          </Button>
-          <Link to={routesPaths.login}>{t('signup.alreadyHaveAccount')}</Link>
-        </div>
-      </form>
+    <div className="container">
+      <div className="form">
+        <form onSubmit={handleSubmit(onSubmit)} noValidate>
+          <h1>{t('signup.title')}</h1>
+          <label htmlFor="name">{t('signup.labels.name')}</label>
+          <Input
+            className="input"
+            register={register}
+            type="text"
+            name="name"
+            error={errors.name}
+            handleFocus={handleFocus}
+          />
+
+          <label htmlFor="email">{t('signup.labels.email')}</label>
+          <Input
+            register={register}
+            type="email"
+            name="email"
+            error={errors.email}
+            handleFocus={handleFocus}
+          />
+
+          <label htmlFor="password">{t('signup.labels.password')}</label>
+          <Input
+            register={register}
+            type="password"
+            name="password"
+            error={errors.password}
+            handleFocus={handleFocus}
+            placeholder="MIN. 6 CHARACTERS LONG"
+          />
+
+          <label htmlFor="password">{t('signup.labels.passwordConfirmation')}</label>
+          <Input
+            register={register}
+            type="password"
+            name="passwordConfirmation"
+            error={errors.passwordConfirmation}
+            handleFocus={handleFocus}
+          />
+
+          <label htmlFor="gender">{t('signup.labels.gender')}</label>
+          <Select
+            name="gender"
+            options={['female', 'male', 'other']}
```

Comment:
Maybe you could move these options to a constant, or even better, to a constants file. This way if you need to reuse them somewhere else (even though in this case is unlikely), you don't have to redefine them.